packages/website/.eslintrc.js

```diff
@@ -4,12 +4,13 @@ module.exports = {
   extends: [
     'molindo/typescript',
     'molindo/react',
-    'plugin:@next/next/recommended'
+    'plugin:@next/next/recommended',
+    "prettier"
```

Comment:
Prettier is already added [via `molindo/typescript`](https://github.com/molindo/eslint-config-molindo/blob/539c8f16049a120a4b0b7ee900b2a9738d386f34/javascript.js#L74-L77).

Can you revert this?